nodejs/src/index.js

```diff
@@ -22,9 +26,18 @@ exports.write = function (struct, compression, callback) {
 };
 
 exports.read = function (Model, rawData, callback) {
+	var callback = arguments.length === 4 ? arguments[3] : arguments[2];
```

Comment:
I'm happy to have a breaking change

```js
exports.read = function (Model, rawData, compression, callback) {}
```

`compression` is the same argument as `write`.